time stamp

# tray id
string tray_id

# state of the tray: occupied/free
bool occupied

# name of the package if any is near
# string package
